Modules/Course/classes/Objectives/class.ilCourseObjectivesTableGUI.php

```diff
@@ -120,6 +120,14 @@ protected function fillRow(array $a_set): void
             $this->tpl->setVariable('LM_IMG', ilObject::_getIcon($data['obj_id'], "tiny", $data['type']));
             $this->tpl->setVariable('LM_ALT', $this->lng->txt('obj_' . $data['type']));
 
+            if ($data['online']) {
+                $this->tpl->setVariable('MAT_VAL_ONOFFLINE', $this->lng->txt('online'));
+                $this->tpl->setVariable('MAT_ONOFFLINE_CLASS', 'smallgreen');
```

Comment:
Please avoid, to address the css classes in PHP files. 
Instead you could add two template blocks in tpl.crs_objectives_table_row.html and parse the one which is needed.

Reply:
Suggestions are added.The proposed changes have been implemented